po/messages.pot

```diff
@@ -2717,8 +3226,9 @@ msgctxt "dlgSgCherryPickConfirmation.chk"
 msgid "Append source commit ID to commit message"
 msgstr ""
 
+#| msgid "This will cherry-pick the selected commit into the Working Tree."
 msgctxt "dlgSgCherryPickConfirmation.fur"
-msgid "This will cherry-pick the selected commit into the Working Tree."
+msgid "This will cherry-pick the selected commits into the Working Tree."
```

Comment:
We do have both:

"This will cherry-pick the selected commits into the Working Tree."

and

"This will cherry-pick the selected commit into the Working Tree."

How should we address this?